<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SUAR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_15" stretch="1,3,3,2">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QPushButton" name="StartStop">
            <property name="text">
             <string>Start/Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Reset">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Interwał</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="Spbox_inter">
            <property name="minimum">
             <double>0.050000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lampkaWyrabiania">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnNetwork">
            <property name="text">
             <string>Sieć</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lblSiec">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_12" stretch="1,2,4">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Generator</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>S</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="ustaw_S">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <property name="verticalSpacing">
              <number>9</number>
             </property>
             <item row="1" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>T</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="ustawT">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="value">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Sygnał</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="Sygnal">
                 <item>
                  <property name="text">
                   <string>Skok</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Pros</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Sinusoida</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>A</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="ustawA">
                 <property name="decimals">
                  <number>3</number>
                 </property>
                 <property name="minimum">
                  <double>-99.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.500000000000000</double>
                 </property>
                 <property name="value">
                  <double>1.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>P</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="ustawP">
                 <property name="maximum">
                  <double>1.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.500000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_13" stretch="0,4">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>PID</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <property name="spacing">
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>K</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="ustawKpid">
                 <property name="singleStep">
                  <double>0.500000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.500000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Td</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="ustawTd">
                 <property name="singleStep">
                  <double>0.500000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.200000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Ti</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chboxCalka">
                 <property name="text">
                  <string>Przed sumą</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="ustawTi">
                 <property name="singleStep">
                  <double>0.500000000000000</double>
                 </property>
                 <property name="value">
                  <double>5.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Pamięć</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pidReset">
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_14" stretch="1,2">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>ARX</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="edytujARX">
             <property name="text">
              <string>Modyfikuj</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,2">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QChartView" name="Chartwartosci"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QChartView" name="ChartUchyb"/>
        </item>
        <item>
         <widget class="QChartView" name="ChartSterowanie"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1189</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
